reporters/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@umaprotocol/reporters",
+  "version": "1.0.0",
+  "description": "Displays Expiring MultiParty on-chain analytics",
+  "private": true,
+  "dependencies": {
+    "dotenv": "^6.2.0",
+    "sinon": "^9.0.2",
+    "winston": "^3.2.1",
```

Comment:
nit: are these used?

Reply:
WInston is, I forgot to remove sinon however